setup_default_backend.sh

```diff
@@ -51,6 +51,8 @@ function install_backend_python_packages {
     pip install -r "requirements-build.txt" && loggood "Successfully installed pip and setuptools" || logerror "Failed to install pip and setuptools"
     log "Installing dependencies from requirements.txt..."
     pip install -r "requirements.txt" && loggood "Successfully installed the dependencies in requirements.txt" || logerror "Failed to install dependencies in requirements.txt"
+    log "Installing all preview dependencies from requirements-full-preview-generator.txt"
+    pip install -r "requirements-full-preview-generator.txt"  && loggood "Successfully installed the dependencies in requirements-full-preview-generator.txt" || logerror "Failed to install dependencies in requirements-full-preview-generator.txt"
```

Comment:
We could allow people to ignore extra preview generator builders? (feel free to not apply)

```suggestion
    if [ -z "$IGNORE_FULL_PREVIEW_GENERATOR" ]; then
        log "Installing all preview dependencies from requirements-full-preview-generator.txt"
        pip install -r "requirements-full-preview-generator.txt"  && loggood "Successfully installed the dependencies in requirements-full-preview-generator.txt" || logerror "Failed to install dependencies in requirements-full-preview-generator.txt"
    fi
```